apps/site/assets/ts/stop/components/DepartureTimes.tsx

```diff
@@ -299,20 +181,24 @@ const DepartureTimes = ({
   const schedules = schedulesByHeadsign(schedulesForDirection);
   return (
     <>
-      {Object.entries(schedules).map(([headsign, schs]) => {
-        const preds = predictionsByHeadsign[headsign]
-          ? predictionsByHeadsign[headsign]
-          : [];
-        const formattedTimes = toDisplayTime(schs, preds, overrideDate);
+      {Object.entries(schedules).map(([headsign, schedules]) => {
+        const predictions = predictionsByHeadsign[headsign] || [];
         return (
           <div
+            // TODO remove this class name in favor of test ids
             className="departure-row-click-test"
             key={`${headsign}-${route.id}`}
-            onClick={() => onClick(route, directionId, schs)}
-            onKeyDown={() => onClick(route, directionId, schs)}
+            onClick={() => onClick(route, directionId, schedules)}
+            onKeyDown={() => onClick(route, directionId, schedules)}
```

Comment:
question: Is this change intentional? Seems like it may have been a merge conflict resolution issue